ROMFS/px4fmu_common/init.d/rc.thermal_cal

```diff
@@ -0,0 +1,56 @@
+#!nsh
+#
+# Thermal Calibration startup script.
+#
+# NOTE: environment variable references:
+#    If the dollar sign ('$') is followed by a left bracket ('{') then the
+#    variable name is terminated with the right bracket character ('}').
+#    Otherwise, the variable name goes to the end of the argument.
+#
+#
+# NOTE: COMMENT LINES ARE REMOVED BEFORE STORED IN ROMFS.
+#
+
+#
+# Declare local script variables and initialize default values
+#
+set TEMP_CALIB_ARGS     none
+
+#
+# Check if we should start a thermal calibration
+#
+if param compare SYS_CAL_GYRO 1
+then
+	set TEMP_CALIB_ARGS "-g"
+	param set SYS_CAL_GYRO 0
+fi
+
+if param compare SYS_CAL_ACCEL 1
+then
+	set TEMP_CALIB_ARGS "-a"
+	param set SYS_CAL_ACCEL 0
+fi
+
+if param compare SYS_CAL_BARO 1
+then
+	set TEMP_CALIB_ARGS "-b"
+	param set SYS_CAL_BARO 0
+fi
+
+#
+# Start the send_event module for temperature calibration
+#
+send_event start
```

Comment:
`send_event` is responsible for more than thermal calibration and thus should be started in rcS